AEPServices/Tests/services/MessageMonitorServiceTest.swift

```diff
@@ -37,43 +37,47 @@
         }
 
         func test_isMessageDisplayed_DefaultIsFalse() {
-            let isDisplayed = mockMessageMonitor?.isMessageDisplayed()
+            let isDisplayed = mockMessageMonitor.isMessageDisplayed()
             XCTAssertTrue((isDisplayed == false))
         }
 
         func test_isMessageDislayed_isTrue_whenDislayMessageCalled() {
             mockMessageMonitor?.displayMessage()
-            let display : Bool = mockMessageMonitor?.isMessageDisplayed() == true
+            let display : Bool = mockMessageMonitor.isMessageDisplayed() == true
             XCTAssertTrue(display)
         }
 
         func test_isMessageDislayed_isFalse_whenDismissMessageIsCalled() {
             mockMessageMonitor?.dismissMessage()
-            let display : Bool = mockMessageMonitor?.isMessageDisplayed() == false
+            let display : Bool = mockMessageMonitor.isMessageDisplayed() == false
```

Comment:
This could be rewritten to XCTAssertFalse(mockMessageMonitor.isMessageDisplayed())